embark-ui/package.json

```diff
@@ -4,7 +4,12 @@
   "private": true,
   "dependencies": {
     "@babel/core": "7.1.0",
+    "@coreui/coreui": "^2.0.15",
```

Comment:
The order was on purpose here, not sure if good idea or not.
But at first it was all the package from the eject of CRA and then our packages.

That makes it easy to upgrade CRA, we don't have to find which package comes from where

Reply:
We can also `git diff` cra's package.json to see if and what changed there to update our stuff when needed.

Reply:
That's fine by me, as said it was maybe not a good idea :)